Form/FormFlowInterface.php

```diff
@@ -27,9 +27,9 @@ function getName();
 	function setFormFactory(FormFactoryInterface $formFactory);
 
 	/**
-	 * @param Request|null $request
+	 * @param RequestStack $requestStack
 	 */
-	function setRequest(Request $request = null);
+	function setRequestStack(RequestStack $requestStack);
```

Comment:
does it really make sense to have such setters in the interface ? You are not using the setter yourselves, so it does not make sense to mandate it. You should let the user decide how they want to implement the access to the request (I would inject it in the constructor)